pandas/tests/frame/methods/test_shift.py

```diff
@@ -656,3 +656,12 @@ def test_shift_axis1_many_periods(self):
 
         shifted2 = df.shift(-6, axis=1, fill_value=None)
         tm.assert_frame_equal(shifted2, expected)
+
+    def test_shift_with_offsets_freq(self):
+        df = DataFrame({"x": [1, 2, 3]}, index=date_range("2000", periods=3))
+        shifted = df.shift(freq="1MS")
+        expected = DataFrame(
+            {"x": [1, 2, 3]},
+            index=date_range(start="02/01/2000", end="02/01/2000", periods=3),
```

Comment:
this is fine here, but in the future, I'd suggest using `2000-02-01` format, it's less ambiguous, whereas `02/01/2000` means different things in different parts of the world

Reply:
Thank you, I will use `2000-02-01` format.